server/app.js

```diff
@@ -0,0 +1,27 @@
+require('dotenv').config()
+import { MongoClient, assert, Twitter, fs} from '../global_imports.js'
+
+let url = 'mongodb://localhost:27017/myproject'
+
+let client = new Twitter({
+  consumer_key: process.env.TWITTER_CONSUMER_KEY,
+  consumer_secret: process.env.TWITTER_CONSUMER_SECRET,
+  access_token_key: process.env.TWITTER_ACCESS_TOKEN,
+  access_token_secret: process.env.TWITTER_ACCESS_SECRET
+})
+// Use connect method to connect to the server
+MongoClient.connect(url, function(err, db) {
```

Comment:
Since you started this file with es6 syntax, I would continue that style throughout the file you are writing.